packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -1087,6 +1087,8 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             { id: 'list.hoverBackground', defaults: { dark: '#2A2D2E', light: '#F0F0F0' }, description: 'List/Tree background when hovering over items using the mouse.' },
             { id: 'list.hoverForeground', description: 'List/Tree foreground when hovering over items using the mouse.' },
             { id: 'list.filterMatchBackground', defaults: { dark: 'editor.findMatchHighlightBackground', light: 'editor.findMatchHighlightBackground' }, description: 'Background color of the filtered match.' },
+            { id: 'tree.indentGuidesStrokeActive', defaults: { dark: '#585858', light: '#a9a9a9', hc: '#a9a9a9' }, description: "Tree Widget's stroke color for active indent guides." },
```

Comment:
where did you get this colors? from VS Code? If not they should be based on VS Code colors

Reply:
They're from VS Code: https://github.com/microsoft/vscode/blob/eb4a91f7eeb30a96057bcc5f8934ac14171c74dd/src/vs/platform/theme/common/colorRegistry.ts#L373

Reply:
VS Code uses `tree.indentGuidesStroke` as  the `id`. Are we OK with `tree.indentGuidesStrokeActive`?

Reply:
@kittaakos I updated the `id` to `tree.indentGuidesStroke`.